themes/SuiteP/css/suitep-base/sidebar.scss

```diff
@@ -27,124 +27,15 @@
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Create_Person_Form.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Create_Security_Group {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Create_Security_Group.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Create_Security_Group.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Import {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Import.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Import.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Import_Line_Item {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Import_Line_Item.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Import_Line_Item.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-List_Maps {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/List_Maps.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/List_Maps.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Quick_Radius_Map {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Quick_Radius_Map.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Quick_Radius_Map.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Resource_Chart {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Resource_Chart.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Resource_Chart.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Role_Management {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Role_Management.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Role_Management.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Schedule_Call {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Schedule_Call.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Schedule_Call.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Schedule_Meeting {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Schedule_Meeting.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Schedule_Meeting.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Security_Groups {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Security_Groups.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Security_Groups.svg");
-  }
-
-  .actionMenuSidebar li a .side-bar-Security_Suite_Settings {
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Security_Suite_Settings.png");
-    background-image: url("../../../../../themes/SuiteP/images/sidebar/Security_Suite_Settings.svg");
-  }
+  .actionMenuSidebar li a .side-bar-action-icon {
```

Comment:
its empty

Reply:
I have when through all my changes to look for the same issue, and I have removed the empty ones.